lib/scavenger_hunt/app/assets/stylesheets/scavenger_hunt/components/_typography.sass

```diff
@@ -38,3 +39,6 @@ p
   font-size: $font-size-small
   font-weight: 600
   text-transform: uppercase
+
+a
+  line-height: 1.35
```

Comment:
This fixes the issue with the `border-bottom` touching the top of the letters of the row beneath it.

Reply:
Let's use the $line-height-ratio defined `application.sass` so we keep this consistent

Reply:
@flipsasser OH DUH! Thanks. Just committed the fix.